02-collections/hw-02-collections.py

```diff
@@ -0,0 +1,47 @@
+import random
+import string
+from collections import defaultdict
+
+# create empty list
+list_of_dicts = []
+
+# randomize number of dicts
+for d in range(random.randint(2, 10)):
+    # set size of dict
+    size = random.randint(2, 10)
+    # define key
+    keys = random.sample(string.ascii_lowercase, size)
+    # define value
+    values = (random.randint(0, 100) for d in range(size))
+    # create dict
+    new_dict = dict(zip(keys, values))
+    # add dict to list
+    list_of_dicts.append(new_dict)
```

Comment:
The same result can be achieved in one line using list comprehension:

list_of_dicts = [{choice(ascii_lowercase): randrange(0, 99) for j in range(randrange(0, 25))}
                    for i in range(randrange(1, 9))]